src/PluginsManager/Microsoft.Performance.Toolkit.Plugins.Runtime/Common/IRepository`2.cs

```diff
@@ -6,7 +6,7 @@
 using System.Threading.Tasks;
 using System.Threading;
```

Comment:
I would consider renaming this to something more appropriate now like `IKeyedRepository`